utils/helpers_test.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetDateDaysDifference(t *testing.T) {
+	created := 1700238956
+
+	testDate := time.Now()
+	days := GetDateDaysDifference(int64(created), &testDate)
+	assert.NotEqual(t, 0, days)
```

Comment:
can we add an assertion for `Equal` and try and do some edge cases aswell like if the date is negative or malformed

Reply:
otherwise its good to see some tests haha